src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -0,0 +1,164 @@
+import './CreateCourse.css';
+
+import Input from '../../common/Input/Input';
+import Textarea from '../../common/Textarea/Textarea';
+import Button from '../../common/Button/Button';
+import CreateAuthor from './components/CreateAuthor/CreateAuthor';
+import AuthorItem from './components/AuthorItem/AuthorItem';
+
+import {
+	buttonNames,
+	placeholders,
+	labels,
+	mockedAuthorsList,
+} from '../../constants';
+
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useNavigate } from 'react-router-dom';
+
+import { formatDuration, formatDate } from '../../helpers/durationFormatter';
+
+const CreateCourse = () => {
+	const navigate = useNavigate();
+
+	const [authors, setAuthors] = useState(mockedAuthorsList);
+	const [courseAuthors, setCourseAuthors] = useState([]);
+	const [newCourse, setNewCourse] = useState({
+		id: uuidv4(),
+		title: '',
+		description: '',
+		creationDate: formatDate(new Date()),
+		duration: 0,
+		authors: [],
+	});
+
+	const addNewAuthor = (author) => {
```

Comment:
Pressing **Create author** button does not create author. It submits course creation and redirects to /courses.

https://github.com/VladimirSadovoi/ReactCourses/assets/157737324/2a4a3440-ba1e-4ae7-aa6a-179fd44678c0

Reply:
fixed, for now on Create author button click, new author is added to Authors list.